pkgs/development/libraries/unicorn-emu/default.nix

```diff
@@ -1,18 +1,25 @@
-{ stdenv, fetchurl, pkgconfig, python }:
+{ stdenv, fetchurl, pkgconfig, python, cmocka, hexdump, setuptools }:
 
 stdenv.mkDerivation rec {
   pname = "unicorn-emulator";
-  version = "1.0.1";
+  version = "1.0.2-rc4";
 
   src = fetchurl {
     url    = "https://github.com/unicorn-engine/unicorn/archive/${version}.tar.gz";
-    sha256 = "0z01apwmvhvdldm372ww9pjfn45awkw3m90c0h4v0nj0ihmlysis";
+    sha256 = "05w43jq3r97l3c8ggc745ai8m5l93p1b6q6cfp1zwzz6hl5kifiv";
   };
 
   configurePhase = '' patchShebangs make.sh '';
   buildPhase = '' ./make.sh '' + stdenv.lib.optionalString stdenv.isDarwin "macos-universal-no";
   installPhase = '' env PREFIX=$out ./make.sh install '';
 
+  NIX_CFLAGS_COMPILE = "-Wno-error";
+  doCheck = true;
+  checkPhase = ''
```

Comment:
if I am correct this is the default. See `checkPhase` in `setup.sh`.

Reply:
That's what the documentation said as well, but using `checkTarget = "test"` didn't work. I'll look into it.

Reply:
I found it, that default only works with the default `buildPhase`, as it sets `foundMakefile` in `setup.sh`.